src/linear_algebra/finsupp.lean

```diff
@@ -5,7 +5,8 @@ Author: Johannes Hölzl
 
 Linear structures on function with finite support `α →₀ M`.
 -/
-import algebra.monoid_algebra
+import data.finsupp.basic
+import linear_algebra.basic
 
```

Comment:
Do you mind adding a module doc string and re-flowing the lines that are too long?

Reply:
The line in your suggestion was the only line >100 chars. I can add a docstring tomorrow.

Reply:
Ah, sorry. I was looking at the [list of exceptions](https://github.com/leanprover-community/mathlib/blob/master/scripts/copy-mod-doc-exceptions.txt#L1318). Possibly some will appear after you merge master?

Reply:
Ah, you're talking about `linear_algebra/finsupp`. Then this discussion should move to  #4597 .